src/syntax/Parser.js

```diff
@@ -4505,6 +4514,14 @@ export class Parser {
   }
 
   reportReservedIdentifier_(token) {
-    this.reportError_(token, `${token.type} is a reserved identifier`);
+    this.reportError_(token.location, `${token.type} is a reserved identifier`);
```

Comment:
I can undo this change and make another PR that fixes all calls to always pass in a location.